pkg/test/resourcefixture/testdata/basic/bigtable/v1beta1/bigtableappprofile/_generated_object_bigtableappprofile.golden.yaml

```diff
@@ -31,4 +31,5 @@ status:
     status: "True"
     type: Ready
   externalRef: projects/${projectId}/instances/profiledep${uniqueId}/appProfiles/bigtableappprofile-${uniqueId}
+  name: projects/${projectId}/instances/profiledep${uniqueId}/appProfiles/bigtableappprofile-${uniqueId}
```

Comment:
To clarify, this file is generated by the test. I was not requesting a manual fix to re-add the field. Instead, we should update the controller to ensure the field is not missing after the test runs.